src/Symfony/Component/Cache/Tests/Adapter/RedisAdapterTest.php

```diff
@@ -98,7 +98,7 @@ public function provideValidSchemes(): array
     {
         return [
             ['redis'],
-            ['rediss'],
+            //['rediss'],
```

Comment:
I don't know how to have a Redis server with TLS. Do somebody know? I thought with `bitnami/redis:6.0` and Docker parameters: `-p 6389:6389 -e REDIS_TLS_ENABLED=yes -e REDIS_TLS_PORT=6389 -e ALLOW_EMPTY_PASSWORD=yes` but it must provide a X.509 certificate, private key and CA X.509 certificate.